meta-rpi-domd/recipes-kernel/linux/files/0001-aarch64-dts-Add-device-tree-for-xen.patch

```diff
@@ -0,0 +1,67 @@
+From 51401aafbdbf3b29b6deecb0398b93c9fe04e716 Mon Sep 17 00:00:00 2001
+From: Dmytro Semenets <dmytro_semenets@epam.com>
+Date: Wed, 17 Apr 2024 23:48:40 +0300
+Subject: [PATCH] aarch64: dts: Add device tree for xen
+
+Device tree used by xen bcm2712-rpi-5-b.dts. Made follow changes:
+ - Add node for xen
+ - Delete nodes uart2 and uart5 because they used same IRQ
+ - Add bootargs for xen
+ - Add bootargs for dom0
+Add device tree for domd
+
+Signed-off-by: Dmytro Semenets <dmytro_semenets@epam.com>
+---
+ arch/arm64/boot/dts/broadcom/Makefile            |  2 ++
+ .../dts/broadcom/bcm2712-raspberrypi5-domd.dts   |  2 ++
+ arch/arm64/boot/dts/broadcom/bcm2712-rpi-5-b.dts | 16 ++++++++++++++++
+ 3 files changed, 20 insertions(+)
+ create mode 100644 arch/arm64/boot/dts/broadcom/bcm2712-raspberrypi5-domd.dts
+
+diff --git a/arch/arm64/boot/dts/broadcom/Makefile b/arch/arm64/boot/dts/broadcom/Makefile
+index 6dd8659a7d8a..a5bf8623194f 100644
+--- a/arch/arm64/boot/dts/broadcom/Makefile
++++ b/arch/arm64/boot/dts/broadcom/Makefile
+@@ -21,6 +21,8 @@ dtb-$(CONFIG_ARCH_BCM2835) += bcm2710-rpi-cm3.dtb
+ dtb-$(CONFIG_ARCH_BCM2835) += bcm2711-rpi-cm4.dtb
+ dtb-$(CONFIG_ARCH_BCM2835) += bcm2711-rpi-cm4s.dtb
+ dtb-$(CONFIG_ARCH_BCM2835) += bcm2712-rpi-5-b.dtb
++dtb-$(CONFIG_ARCH_BCM2835) += bcm2712-raspberrypi5-xen.dtb
++dtb-$(CONFIG_ARCH_BCM2835) += bcm2712-raspberrypi5-domd.dtb
+ dtb-$(CONFIG_ARCH_BCM2835) += bcm2712d0-rpi-5-b.dtb
+ dtb-$(CONFIG_ARCH_BCM2835) += bcm2712-rpi-cm5-cm5io.dtb
+ dtb-$(CONFIG_ARCH_BCM2835) += bcm2712-rpi-cm5-cm4io.dtb
+diff --git a/arch/arm64/boot/dts/broadcom/bcm2712-raspberrypi5-domd.dts b/arch/arm64/boot/dts/broadcom/bcm2712-raspberrypi5-domd.dts
+new file mode 100644
+index 000000000000..1457e696f968
+--- /dev/null
++++ b/arch/arm64/boot/dts/broadcom/bcm2712-raspberrypi5-domd.dts
+@@ -0,0 +1,2 @@
++// SPDX-License-Identifier: GPL-2.0
++#include "arm/broadcom/bcm2712-rpi-5-b.dts"
```

Comment:
why do you add chosen to domd device tree? 
Why not use prod-devel approach with xen-chosen.dtsi?

Reply:
using this approach will allow you to make changes to image size on build and do not do this on each boot